src/process.rs

```diff
@@ -2,12 +2,14 @@ use anyhow::{anyhow, Result};
 use flate2::read::GzDecoder;
 use indicatif::{ProgressBar, ProgressStyle};
 use nohash::IntMap;
-use raad::le::*;
+use raad::le::{R, W};
+use serde::Deserialize;
```

Comment:
youve changed serde_derive::Deserialize to serde::Deserialize-- iirc serde_derive::Deserialize compiles faster
```suggestion
use serde_derive::Deserialize;
```

Reply:
ye but it removes the need to depend on serde-derive directly, I guess i can revert this back since there's not much different